exo-go/src/cmd/clean.go

```diff
@@ -23,56 +21,19 @@ var cleanCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		err = removeImages(c)
+		err = dockerHelpers.RemoveDanglingImages(c)
```

Comment:
Theses are related as she extracted some functions from this file and moved them into a helper function as she needed them somewhere else